scss/_settings.scss

```diff
@@ -44,12 +44,15 @@ $font__pre: "Courier 10 Pitch", Courier, monospace;
 $font__line-height-body: 1.5;
 $font__line-height-pre: 1.6;
 
+$font_body: "Raleway", sans-serif, Arial, Helvetica, sans-serif;
+
 /*--------------------------------------------------------------
 ## Font Size
 --------------------------------------------------------------*/
-$font-size-h1: 64px;
-$font-size-h2: 56px;
-$font-size-body: 16px;
+$font_size_h1: 64px;
```

Comment:
I reccomend creating more generalized var names because you can use them for a variety of different classes. for example: 
```
$font-x-large: 64px; 
$font-large: 56px; 
$font-medium: 18px; 
$font-standard: 16px; 
.... 
```
This will allow you to intelligently think of how many sizes you want overall for your site then stick to it for all your text styles.